docs/usage/configuration-options.md

```diff
@@ -318,15 +318,17 @@ Add config here if you wish it to apply to Docker package managers Dockerfile an
 
 ## draftPR
 
-If you want the PRs created by Renovate to be considered as drafts rather than normal PRs in Github you could add this property to your `renovate.json`:
+If you want the PRs created by Renovate to be considered as drafts rather than normal PRs, you could add this property to your `renovate.json`:
 
 ```json
 {
   "draftPR": true
 }
 ```
 
-Please see [draft pull requests on Github](https://github.blog/2019-02-14-introducing-draft-pull-requests/) for more information about draft PRs.
+This option is only supported on the following platforms: GitHub, GitLab, Azure
+
+Note that GitLab implements draft status by checking whether the PR's title starts with certain strings. Therefore, draftPR on GitLab is incompatible with the legacy method of triggering Renovate to rebase a PR by renaming the PR to start with `rebase!`.
```

Comment:
```suggestion
Note that GitLab implements draft status by checking whether the PR's title starts with a certain string. Therefore, draftPR on GitLab is incompatible with the legacy method of triggering Renovate to rebase a PR by renaming the PR to start with `rebase!`.
```

I'm not sure, just throwing this out there: Is it necessary for GitLab to check only 1 string or does it check multiple strings to determine if this should be a draft PR?

Reply:
Per GitLab docs, "WIP:" is supported until 14.0 but is deprecated after 13.2 in favor of "Draft:", so yes, we do need to check for both strings.

Reply:
Okay, just wanted to check if the documentation was accurate. I'll mark this as resolved then. :smile: